sandbox.go

```diff
@@ -36,6 +36,8 @@ type Sandbox interface {
 	SetKey(key string) error
 	// Delete destroys this container after detaching it from all connected endpoints.
 	Delete() error
+	// Retrieve the interfaces in the sandbox, for restoring checkpointed container
+	Interfaces() []osl.Interface
```

Comment:
though `Interface` is exposed as an osl interface, it feels odd to include it as part of the API.
osl is purely an internal implementation that provides os layer abstraction away from the clients.
Having this interface exposed via Interfaces() API which gives more control over life-cycle of the interface such as `Remove` makes me uncomfortable.

Lets hear from @mrjana on his opinion.